tests/test_backends.py

```diff
@@ -76,6 +77,31 @@ def test_init_fails_for_rs_algorithms_when_crypto_not_installed(self):
             ):
                 TokenBackend(algo, "not_secret")
 
+    def test_jwk_client_not_available(self):
+        import builtins
+
+        from rest_framework_simplejwt import backends
+
+        def myimport(name, globals=None, locals=None, fromlist=(), level=0):
+            if name == "jwt" and fromlist == ("PyJWKClient", "PyJWKClientError"):
+                raise ImportError
+            return realimport(name, globals, locals, fromlist, level)
+
+        realimport = builtins.__import__
```

Comment:
possibility of failure during assertions, so cleanup needs to be done better. Perhaps in setup, you can set `realimport = builtins.__import__`

Reply:
I've update the `setUp()`. Could you please review and correct me if i'm wrong.